selectors.go

```diff
@@ -0,0 +1,8 @@
+package gocrawl
+
+import "github.com/andybalholm/cascadia"
+
+var (
+	aHrefMatcher    = cascadia.MustCompile("a[href]")
+	baseHrefMatcher = cascadia.MustCompile("base[href]")
```

Comment:
Let's move them to `worker.go`, not worth creating a new file just for that, and that's where they are used. I'd put them just above the `processLinks` func, to keep them close to use.

Reply:
Fine